### PR TITLE
Address release comments

### LINKED ISSUE #22: Address Tim's Comments
Tim would like the following addressed: 
1. Also the diagramType should be set in the umpleonline URL; in other words  it should have the following at the end of the UmpleOnline URL
   &diagramtype=GvClass

or if State Diagram is selected in the tool then GvState

Can both these change be made; then I will put it online?

2.5 It would also be nice if there was one explanatory line at the top:
"A tool to convert models in open repositories and generate Umple, from which you can generate code and diagrams in many formats"

and 2.75 -- if the URL could say http://try.umple.org instead of the cruise URL that would allow for portability.

### PR DESCRIPTION
**Pull request by Nava2**
- Removes octicons
- Add more information in the about section 
  screen shot 2015-04-08 at 10 13 10
- Add diagram type to the umple online links, closing #23 and #21
  e.g. `http://try.umple.org/?filename=localhost/data/umpr_repos/AtlanZooEcore/WTIP_SimpleRDBMS.ecore.ump&diagramtype=GvClass`
- Changed the Umple URL to be try.umple.org (see previous point)

Closes #22.

### CODE REVIEW

**Comment by Nava2** [comment]
Oops, also adds a subtitle:
screen shot 2015-04-08 at 10 15 04

File: app/views/home.html.php [hunk 1]
```diff
@@ -171,10 +197,10 @@ class="btn btn-sm btn-danger">
                   <button class="btn btn-danger text-center status-badge status-badge-failed"
                           type="button"
                           data-toggle="collapse"
-                          data-target="#message-row-<?php echo $idTag ?>"
+                          data-target="#message-row-<?= $idTag ?>"
```
Comment (aforward) [inline app/views/home.html.php:200]: <?php echo please

File: app/views/layout/html_head.html.php [hunk 2]
```diff
@@ -16,7 +16,6 @@
 <link href="/scripts/solid/css/style.css" rel="stylesheet">
 <link href="/scripts/solid/css/font-awesome.min.css" rel="stylesheet">
 <link href="/scripts/solid_helper.css" rel="stylesheet">
-<link rel="stylesheet" href="/octicons/octicons/octicons.css">
```
Comment (aforward) [inline app/views/layout/html_head.html.php:19]: don't need?

**Comment by aforward** [comment]
Address or ignore my comments and merge

File: app/views/layout/html_head.html.php (hunk 2 above)
Comment (Nava2) [inline app/views/layout/html_head.html.php:19]: I didn't realize that SOLID brings in Font Awesome, it has the icon I wanted.

**Pull request merged**